crates/burn-import/src/onnx/op_configuration.rs

```diff
@@ -120,6 +120,21 @@ pub fn max_pool2d_config(curr: &Node) -> MaxPool2dConfig {
         .with_padding(padding)
         .with_dilation([dilations[0] as usize, dilations[1] as usize])
 }
+pub fn prelu_config(curr: &Node) -> PReluConfig {
+    let mut alpha = 0.01;
+    let mut num_parameters = 0;
+    for (key, value) in curr.attrs.iter() {
+        match key.as_str() {
+            "alpha" => alpha = value.clone().into_f32(),
+            "num_parameters" => num_parameters = value.clone().into_i32(),
+            _ => {}
+        }
+    }
```

Comment:
ONNX Spec does not have alpha nor num_parameters, so they should be removed: https://onnx.ai/onnx/operators/onnx__PRelu.html

I recommend you would remove prelu_config function and just use PReluConfig::new() with it's defaults. You do not need to store or pass it.

config is meant to extract attributes (such as [Conv attributes](https://onnx.ai/onnx/operators/onnx__Conv.html#attributes))  as Module configuration.

You can also see the ONNX example you checked in does not contain alpha and num_params:

<img width="911" alt="image" src="https://github.com/tracel-ai/burn/assets/939125/7e1ffe29-3563-498e-90b2-bfc5bde80965">